gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobInfo.java

```diff
@@ -441,23 +441,49 @@ public static Builder builder(String bucket, String name) {
   }
 
   static BlobInfo fromPb(StorageObject storageObject) {
-    Builder builder = new Builder()
-        .bucket(storageObject.getBucket())
-        .cacheControl(storageObject.getCacheControl())
-        .contentEncoding(storageObject.getContentEncoding())
-        .crc32c(storageObject.getCrc32c())
-        .contentType(storageObject.getContentType())
-        .generation(storageObject.getGeneration())
-        .md5(storageObject.getMd5Hash())
-        .mediaLink(storageObject.getMediaLink())
-        .metageneration(storageObject.getMetageneration())
-        .name(storageObject.getName())
-        .contentDisposition(storageObject.getContentDisposition())
-        .componentCount(storageObject.getComponentCount())
-        .contentLanguage(storageObject.getContentLanguage())
-        .etag(storageObject.getEtag())
-        .id(storageObject.getId())
-        .selfLink(storageObject.getSelfLink());
+    Builder builder = new Builder().bucket(storageObject.getBucket()).name(storageObject.getName());
```

Comment:
Basically because of the following behaviour:
```
BucketInfo bucketInfo = BucketInfo.of("b");
bucketInfo.equals(BucketInfo.fromPb(bucketInfo.toPb())); // False
```
Which I think is quite undesirable.